common/src/main/kotlin/io/homeassistant/companion/android/common/data/integration/impl/IntegrationRepositoryImpl.kt

```diff
@@ -684,11 +684,13 @@ class IntegrationRepositoryImpl @AssistedInject constructor(
             try {
                 return action(url)
             } catch (e: Exception) {
+                Timber.w(e, "Failed $requestName on $url")
```

Comment:
This may contain sensitive information (webhook url/ID) so shouldn't be logged in production

Reply:
Indeed! It should only be the requestName